src/users/users.module.ts

```diff
@@ -2,8 +2,18 @@ import { Logger, Module } from '@nestjs/common';
 import { UsersRepository } from './users.repository';
 import { UsersService } from './users.service';
 import { UsersController } from './users.controller';
+import { MailModule } from '../mail/mail.module';
+import { JwtModule } from '@nestjs/jwt';
+import { jwtConstants } from '../auth/guards/constants';
 
 @Module({
+  imports: [
+    MailModule,
```

Comment:
please remove MailModule in this line as well as line 5. Other than that, looks good to me!